attributes/default.rb

```diff
@@ -97,13 +97,24 @@
 default['openldap']['slapd_provider'] = nil
 default['openldap']['slapd_replpw'] = nil
 default['openldap']['slapd_rid'] = 102
-default['openldap']['syncrepl_interval'] = '01:00:00:00'
-default['openldap']['syncrepl_type'] = 'refreshAndPersist'
-default['openldap']['syncrepl_filter'] = '(objectClass=*)'
-default['openldap']['syncrepl_use_tls'] = 'no' # yes or no
```

Comment:
you should probably maintain some backcompat for awhile at least.

unfortunately that seems like it'll be messy since you'll likely wind up setting the default value of e.g. `default['openldap']['syncrepl_config']['type'] = nil` and then doing `syncrepl_config_type = node['openldap']['syncrepl_config']['type'] || node['openldap']['syncrepl_type']` (again in recipe code set plain old ruby vars rather than setting attrs).